health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/FacilityCatchmentMappnig.js

```
@@ -0,0 +1,22 @@
+import { Header, InboxSearchComposer } from "@egovernments/digit-ui-react-components";
+import React, { useState, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import facilityMappingConfig from "../../configs/FacilityMappingConfig";
+
+const FacilityCatchmentMapping = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)